sdk/src/cameras/ad-fxtof1-ebz/camera_fxtof1.cpp

```diff
@@ -188,13 +189,15 @@ aditof::Status CameraFxTof1::initialize() {
 }
 
 aditof::Status CameraFxTof1::start() {
-    // return m_depthSensor->start(); // For now we keep the device open all the time
-    return aditof::Status::OK;
+    return m_depthSensor
+        ->start(); // For now we keep the device open all the time
```

Comment:
The comments no longer make sense if you uncommented this line.

Reply:
Fixed.